modules/stanford_bean_types_featured/scss/stanford_bean_types_featured.scss

```diff
@@ -44,9 +44,123 @@ div[class*="featured"] {
   }
 }
 
-.span6 .featured-block {
-  margin-left: 0;
-  margin-right: 0;
+.front {
+  .fullwidth {
+    .paragraphs-item-p-featured {
+      &.has-image {
+        .group-headline-h2 {
+          margin-top: .2em;
+          margin-bottom: 20px;
+        }
+      }
+    }
+  }
+}
+
+.front .fullwidth .paragraphs-item-p-featured {
```

Comment:
Can this be combined with the rule above? They have the same handful of wrappers.